App/arca/components/sidebar.jsx

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import { Link } from "react-router-dom"; 
+import { useState } from "react";
+import "../node_modules/bootstrap/dist/css/bootstrap.min.css";
+import "../assets/css/components.scss";
+import "../assets/css/helpCenter.scss";
+import "../assets/css/main.scss";
+import menuimg from "../assets/img/icons/menu-burger.svg";
+import arcalogo from "../assets/img/icons/logo.svg";
+import dashboardimg from "../assets/img/icons/ico_apps-font_vio}.svg";
+import estetoscopioimg from "../assets/img/icons/ico_stethoscope.svg";
+import usersimg from "../assets/img/icons/users-alt-free-icon-font.svg";
+import documentimg from "../assets/img/icons/ico_document-signed.svg";
+import userprofile from "../assets/img/icons/ico_user-free-icon-font (1).svg";
+import help from "../assets/img/icons/help.svg";
+
+
+function Sidebar(){
+  const [estadoMenu, setEstadoMenu] = useState(false);
+  function despSidebar() {
+    const conDash = document.querySelector('.container-dashboard');
+    const sidebar = document.querySelector('.body-dashboard');
+    const btnSideBar = document.querySelector('.btn-des-sidebar');
+    const imgBtn = document.querySelector('.img-btn');
+    const itemHelp = document.querySelector('.item-help');
+    const conHelp = document.querySelector('.con-help-center');
+    const imgArca = document.getElementById('arca-logo');
+
+    if (sidebar.clientWidth === 70) {
+      setTimeout(() => {
+        conDash.style.gridTemplateColumns = "260px 1fr";
+        sidebar.style.width = "260px";
+        btnSideBar.style.left = "245px";
+        imgBtn.src = "../assets/img/icons/angle-double-small-right-free-icon-font.svg";
+        imgArca.style.width = "85.5px";
+        imgArca.style.height = "35px";
+        conHelp.style.display = "grid";
+        itemHelp.style.display = "none";
+      }, 500);
+    } else {
+      setTimeout(() => {
+        conDash.style.gridTemplateColumns = "70px 1fr";
+        sidebar.style.width = "70px";
+        btnSideBar.style.left = "55px";
+        imgBtn.src = "../assets/img/icons/angle-double-small-left-free-icon-font.svg";
+        imgArca.style.width = "30px";
+        imgArca.style.height = "35px";
+        conHelp.style.display = "none";
+        itemHelp.style.display = "grid";
+      }, 500);
+    }
+  }
```

Comment:
Avoid updating the styles or component behavior through javascript, instead of that you can use Component variables and use them where you need it... but for this case I think a media query could me enough